p2p/libp2p/networksharding/listsSharder.go

```diff
@@ -16,18 +17,19 @@ import (
 	kbucket "github.com/libp2p/go-libp2p-kbucket"
 )
 
-var _ p2p.CommonSharder = (*listsSharder)(nil)
+var _ p2p.Sharder = (*listsSharder)(nil)
 
 const minAllowedConnectedPeersListSharder = 5
 const minAllowedValidators = 1
 const minAllowedObservers = 1
 const minUnknownPeers = 1
 
 const intraShardValidators = 0
-const intraShardObservers = 1
-const crossShardValidators = 2
-const crossShardObservers = 3
-const unknown = 4
+const intraShardObservers = 10
```

Comment:
these are used like enums, right?

Reply:
Yes, as to easily add the next ones (FullHistoryObservers, for example)

Reply:
ok